src/components/intent.ts

```diff
@@ -520,7 +520,7 @@ export class Intent {
      * @return A Promise that resolves with the requested user's profile
      * information
      */
-    public async getProfileInfo(userId: string, info: string, useCache=true) {
+    public async getProfileInfo(userId: string, info: "avatar_url"|"displayname"|null = null, useCache = true) {
```

Comment:
Are "avatar_url"|"displayname" the only values that `profile.get()` supports?
TypeScript becomes a pain if Types are incorrectly too strict.

But if they are the only reasonable values, great to help bridges using this library to spell them correctly.

Reply:
The spec doesn't allow arbitrary keys, and it's been that way for as long as Matrix has been around. We can update this if the spec changes at some point.